installation_instructions/operation_and_folder_structure.md

```diff
@@ -0,0 +1,98 @@
+
+``` 
+|--- landbosse-YEAR-MONTH-DAY-HOUR-MINUTE-SECOND
+|---|
+|   |--- landbosse-output.xlsx
+|   |--- project_list.xlsx
+|   |--- project_data
+|      |--- project_data_1.xlsx
+|      |--- project_data_2.xlsx
+|      |--- etc.
+|
+|--- landbosse-YEAR-MONTH-DAY-HOUR-MINUTE-SECOND
+|---|
+    |--- landbosse-output.xlsx
+    |--- project_list.xlsx
+    |--- project_data
+       |--- project_data_1.xlsx
+       |--- project_data_2.xlsx
+       |--- etc.
+```
+
+At the top level are directories timestamped with the date and time the model executed.
+
+## Specifying files during LandBOSSE runs
+
+The easiest way to specify paths to LandBOSSE folders is on the command line, as shown below.
+
+``` 
+python main.py --input PATH_TO_INPUT_FOLDER --output PATH_TO_OUTPUT_FOLDER
+```
+
+If you prefer shorter command lines, you have the following option:
+
+``` 
+python main.py -i PATH_TO_INPUT_FOLDER -o PATH_TO_OUTPUT_FOLDER
+```
+
+If you don't want to set the paths every time you execute LandBOSSE, you can set the `LANDBOSSE_INPUT_DIR` and `LANDBOSSE_OUTPUT_DIR` environment variables, but that is not necessary.
+
+Here's a flowchart of how the model gathers and copies input data during normal operation:
+
+![flowchart of validation process](normal-operation-flowchart.png)
+
+## Validating model output
+
+Recall that a LandBOSSE output folder can be used as an input folder. This means that every output folder is a record of inputs, with their associated outputs, that have been created by the LandBOSSE model at a certain point in time. As software development of the model takes place, software defects may be introduced.
+
+The validation functionality of LandBOSSE helps guard against these bugs from creeping in. The idea is simple: If you take an output folder from a time when you **know that the code operated correctly**, you can attempt to recreate those results with another version of the code. If you can reproduce the results, that means the model is still functioning normally. If you don't reproduce the results, that means the model failed and needs to be fixed.
+
+Here is a flowchart of the validation process:
+
+![flowchart of validation process](validation-flowchart.png)
+
+First, the model runs. However, instead of the final output being written as a file, it is compared to the previously calculated LandBOSSE output. If the outputs compare successfully, the model is working the same as it did at the previous point time when it was verified to be working properly.
+
+To run this validation process, you need to have an output folder of LandBOSSE that will be used as the **expected** data. The expected data is the data you know to be true. In contrast, the data being validated are referred to as **actual** data. The command will look something like this:
+
+```
+python main.py --input PATH_TO_PRIOR_LANDBOSSE_OUTPUT --validate
```

Comment:
I was somewhat confused about what the validation file structure and naming convention was. See my commit about updating file paths for validation and revise markdown if you think those changes make sense.